railties/railties.gemspec

```diff
@@ -44,6 +44,7 @@ Gem::Specification.new do |s|
   s.add_dependency "rake", ">= 12.2"
   s.add_dependency "thor", "~> 1.0"
   s.add_dependency "zeitwerk", "~> 2.6"
+  s.add_dependency "irb"
```

Comment:
@st0012 Is there a reasonable base version we want to prefer? Like `>= 1.0.0"` or similar? I know for sure rubygems will warn about these, which is annoying but ignorable -- however we may want to ensure user doesn't accidentally use an old broken version somehow. If that's possible given irb is a default gem? 🤔

Reply:
FYI https://github.com/rails/rails/pull/47442#discussion_r1119398688